Source/init.cpp

```diff
@@ -136,51 +148,30 @@ void init_archives()
 	if (diabdat_mpq == NULL || !SFileOpenFile("ui_art\\title.pcx", &fh))
 		InsertCDDlg();
 	SFileCloseFile(fh);
+	fileinfo.originalarchivefile = diabdat_mpq_path->c_str();
 
+	static std::string *patch_rt_mpq_path = new std::string;
 	patch_rt_mpq = init_test_access(patch_rt_mpq_path, "patch_rt.mpq", "DiabloInstall", 2000, FS_PC);
 	if (patch_rt_mpq == NULL)
 		patch_rt_mpq = init_test_access(patch_rt_mpq_path, "patch_sh.mpq", "DiabloSpawn", 2000, FS_PC);
+	fileinfo.patcharchivefile = patch_rt_mpq_path->c_str();

```

Comment:
actually lets not set this any more, it only serves to leak system info to other players

Reply:
Done